<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>144</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>422</width>
    <height>144</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ZynOS Decrypt | 8ThBiT</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>422</width>
     <height>124</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>422</width>
     <height>124</height>
    </size>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Victim : </string>
    </property>
    <widget class="QLineEdit" name="txtURL">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>http://192.168.1.1/</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnCheck">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>19</y>
       <width>75</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Check</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnFile">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>19</y>
       <width>75</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>From File</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Password</string>
    </property>
    <widget class="QLineEdit" name="txtPassword">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>381</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>txtURL</tabstop>
  <tabstop>btnCheck</tabstop>
  <tabstop>btnFile</tabstop>
  <tabstop>txtPassword</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
